src/main/java/frc/robot/subsystems/DriveFilter.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems;
+
+import frc.robot.core.RobotPose;
+
+/**
+ * Base Drive Filter
+ * @author dcowden
+ */
+public abstract class DriveFilter {
+    private boolean enabled;
+    
+    public void setEnabled(boolean enabled){
```

Comment:
I don't like setEnabled(boolean).  I'd rather see enabled() and disabled() since that's the model that's used in several of the WPI classes (e.g. PIDController).